### PR TITLE
fix: fix breakpoint issue on smaller devices

### PR DESCRIPTION
**Pull request by discodavey**
This is causing the breakpoints issue

### CODE REVIEW

**Comment by davidcmoulton** [comment]
PR title should more describe what problem the PR is solving, rather than how. For example if it turns out a different change is actually what's required to fix the layout at various page widths, then as it stands the PR title would have to change, which could be confusing.

**Comment by discodavey** [comment]
Have you any suggestions?

**Comment by discodavey** [comment]
@davidcmoulton This is the PR I am on about. You have asked me to change the title which I have done. Not sure if you have tested the layout yet without what I have removed or not as you have not commented.

**Comment by davidcmoulton** [comment]
This doesn't fix the issue: the widths at which the layout changes are not the same as on an eLife article, even accounting for the iframe margins.

This PR title mentions smaller screen widths. I'd look at addressing them all in one go otherwise one change may end up having a knock on effect on a previously fixed one.

**Comment by davidcmoulton** [comment]
We want things to work the same here re layout for screen width as it does on eLife. Rather than reimplement eLife's grid system here in the same way, which would be complicated for little benefit, it might be simpler just use media queries to control the width of the article in Thema. As long as the aim is achieved of getting the same article (& article component) width sizing at the same screen widths as eLife, it's okay to implement it differently.

**Comment by discodavey** [comment]
@davidcmoulton Cannot seem to base this to master

**Comment by discodavey** [comment]
Added padding (left and right) for different breakpoints to match eLife website as much as possible. 

Small devices: 7%
Larger devices: 3%

**Comment by davidcmoulton** [comment]
The max line length is too long at larger viewport widths, cf eLife.

(Recommend using dev tools to zero the margins on the `<iframe>` and increase its width to 100% when you test in-browser so it doesn't get in the way).

**Comment by discodavey** [comment]
Margins have now been removed as updated from master and this sorted the issue.

File: src/themes/elife/styles.css [hunk 1]
```diff
@@ -112,3 +111,10 @@ main {
   text-align: center;
   text-transform: capitalize;
 }
+
+@media only screen and (min-width: 45.625em) {
```
Comment (davidcmoulton) [inline src/themes/elife/styles.css:115]: We should take advantage of the [`postcss-custom-media`](https://github.com/postcss/postcss-custom-media) plugin [used by Thema](https://github.com/stencila/thema/blob/f966b5b0f6152e88d179c7639db9aa4f627fc464/postcss.config.js#L12) to eliminate the magic numbers of the breakpoint. Can then use a name derived from the eLife breakpoint names, e.g. this one could be `medium`, derived from https://github.com/elifesciences/pattern-library/blob/1a8e5c72d6c22fbb1d9195b395c55bb7e5dcde78/assets/sass/_variables.scss#L118. Seems okay to set the value in `em` as it has been here.

Comment (discodavey) [reply src/themes/elife/styles.css]: I have changed this to wide as this fits what was 45.625em in terms of how the page changed.

**Comment by discodavey** [comment]
@davidcmoulton ready for review

**Pull request merged**